hubspot/serializers.py

```diff
@@ -22,7 +22,7 @@ class HubspotProductSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = BootcampRun
-        fields = ["title"]
+        fields = ["title", "bootcamp_run_id"]
```

Comment:
Does this mean that the bootcamp_run_id will be shipped to Hubspot? We should definitely test this on RC, in case there's any additional configuration required.

Reply:
There were some additional configurations for that, I've created a PR for that (https://github.com/mitodl/bootcamp-ecommerce/pull/1189), In case you would like to take a look.